content/faq/_index.pl.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Frequently Asked Questions {translate me}
```

Comment:
1. I tried to use translations from `config.toml` writing:

```
title: "{{ trans(key='faq-menu-title', lang='pl') }}"
```

But it didn't work. Looks like we can use `trans()` code only in Markdown body.

2. We can generate `content/faq/_index.##.md` files for all languages. But if there are no taxonomies then Zola doesn't generate `/##/faq/` page and user will see error page:

```
You're seeing this page because we couldn't find a template to render.
```

Reply:
1. No, I meant to insert translations from strings.txt below into each .md file directly.
2. If the main site is built ok, without any references to missing pages, then it's acceptable. Otherwise, let's translate only necessary langs for now and add other translations later.

```
  [faq]
    comment = Button in the main Help dialog
    tags = android,ios
    en = Frequently Asked Questions
    af = Gereelde vrae
    ar = اسئلة مكررة
    az = Tez-tez soruşulan suallar
    be = Пытанні і адказы
    bg = Въпроси и отговори
    ca = Preguntes més freqüents
    cs = Otázky a odpovědi
    da = Spørgsmål og svar
    de = Häufige Fragen und Antworten
    el = Συχνές ερωτήσεις
    es = Preguntas frecuentes
    et = Korduma kippuvad küsimused
    eu = Ohiko galderak
    fa = پرسش‌های زیاد پرسیده شده
    fi = Usein kysytyt kysymykset
    fr = Foire aux questions
    he = שאלות ותשובות
    hi = अक्सर पूछे जाने वाले प्रश्नों
    hu = Gyakran Ismételt Kérdések
    id = Pertanyaan dan jawaban
    it = Domande frequenti
    ja = よくある質問
    ko = 질문과 답변
    lt = Dažniausiai užduodami klausimai
    mr = वारंवार विचारलेले प्रश्न
    nb = Spørsmål og svar
    nl = Vragen en antwoorden
    pl = Pytania i odpowiedzi
    pt = Perguntas frequentes
    ro = Întrebări frecvente
    ru = Вопросы и ответы
    sk = Otázky a odpovede
    sv = Frågor och svar
    sw = Maswali na majibu
    th = คำถามและคำตอบ
    tr = Sıkça Sorulan Sorular
    uk = Питання та відповіді
    vi = Câu hỏi và trả lời
    zh-Hans = 问题和解答
    zh-Hant = 問題和解答

```

Reply:
@strump you can use DeepL as the initial version. And then tag our translate groups on Github (for selected languages only) to review automated translations.

Reply:
Updated FAQ description in `content/faq/_index.##.md` files using DeepL.

But I don't know how to translage 'F.A.Q.' menu item into German. Please review